### PR TITLE
Added gpcp-product-pricing-labels.js and gpcp-product-pricing-labels.php

### PR DESCRIPTION
**Pull request by scyt**
Original gist: https://gist.github.com/spivurno/cc461abccceaf9aec531210cb0846476

### CODE REVIEW

File: gp-conditional-pricing/gpcp-product-pricing-labels.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	function update_price_lables() {
+		$( 'label[data-gpcp-template], option[data-gpcp-template]' ).each( function() {
+			var $priceElem = $( this ).is( 'option' ) ? $( this ) : $( this ).siblings( 'input' ),
+				price = gformFormatMoney( $priceElem.val().split( '|' )[1] ),
+				template = $( this ).attr( 'data-gpcp-template' );
+			$( this ).html( template.replace( '{price}', price ) );
+		} );
+	}
+	gform.addAction( 'gpcp_after_update_pricing', update_price_lables);
+	update_price_lables();
```
Comment (spivurno) [inline gp-conditional-pricing/gpcp-product-pricing-labels.js:17]: Is the `gpcp_after_update_pricing` filter not triggered on the first render? I would have expected it to...

If not, I like your solution. Small type in the function name: "labels" vs "lables".

File: gp-conditional-pricing/gpcp-product-pricing-labels.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * http://gravitywiz.com/documentation/gravity-forms-conditional-pricing/
+ *
+ * This snippet requires the PHP counterpart gpcp-product-pricing-labels.php
+ */
+( function( $ ) {
+	function update_price_lables() {
+		$( 'label[data-gpcp-template], option[data-gpcp-template]' ).each( function() {
+			var $priceElem = $( this ).is( 'option' ) ? $( this ) : $( this ).siblings( 'input' ),
+				price = gformFormatMoney( $priceElem.val().split( '|' )[1] ),
+				template = $( this ).attr( 'data-gpcp-template' );
```
Comment (spivurno) [inline gp-conditional-pricing/gpcp-product-pricing-labels.js:12]: Can we update this so each variable has a `var` and each line ends with a semi-colon?

**Review by spivurno — CHANGES_REQUESTED** [review]
Good stuff, man! A couple changes requested.

File: gp-conditional-pricing/gpcp-product-pricing-labels.js (hunk 1 above)
Comment (scyt) [reply gp-conditional-pricing/gpcp-product-pricing-labels.js:17]: It is, but there was an issue if the Conditional Pricing rule was set to a Date field that was populated using {date_mdy}. 

https://secure.helpscout.net/conversation/1254561365/19350/

File: gp-conditional-pricing/gpcp-product-pricing-labels.js (hunk 2 above)
Comment (scyt) [reply gp-conditional-pricing/gpcp-product-pricing-labels.js]: Sure. Fixed in 053f88d.

**Review by spivurno — APPROVED** [review]
Good stuff. Thanks, Scott!

**Pull request merged**